Vagrantfile

```diff
@@ -65,7 +65,7 @@ Vagrant.configure(2) do |config|
 
     echo "Installing chrome driver..."
     # Install Chromium Driver
-    apt-get install -y chromium-driver  > /dev/null
```

Comment:
I think we need both. I have to use Debian (because I use an M1 Macbook), which does not have a `chromium-chromedriver` package.